metasync.js

```diff
@@ -16,6 +16,7 @@ const submodules = [
   'poolify', // Create pool from factory
   'queue', // Concurrent queue
   'throttle', // Throttling utilities
+  'async-emitter', // AsyncEmitter
```

Comment:
This list is sorted alphabetically since https://github.com/metarhia/metasync/pull/400, thus you should probably add this entry after `'array'`.
Also, you haven't regenerated the documentation, to do this, run `npm run doc`.